functional/lcow_test.go

```diff
@@ -22,33 +22,21 @@ import (
 // TestLCOWUVMNoSCSINoVPMemInitrd starts an LCOW utility VM without a SCSI controller and
 // no VPMem device. Uses initrd.
 func TestLCOWUVMNoSCSINoVPMemInitrd(t *testing.T) {
-	var scsiCount uint32 = 0
-	var vpmemCount uint32 = 0
-	opts := &uvm.OptionsLCOW{
-		Options: &uvm.Options{
-			ID: "uvm",
-		},
-		VPMemDeviceCount:    &vpmemCount,
-		SCSIControllerCount: &scsiCount,
-	}
+	opts := uvm.NewDefaultOptionsLCOW(t.Name(), "")
+	opts.SCSIControllerCount = 0
+	opts.VPMemDeviceCount = 0
+
 	testLCOWUVMNoSCSISingleVPMem(t, opts, `Command line: initrd=/initrd.img`)
 }
 
 // TestLCOWUVMNoSCSISingleVPMemVHD starts an LCOW utility VM without a SCSI controller and
 // only a single VPMem device. Uses VPMEM VHD
 func TestLCOWUVMNoSCSISingleVPMemVHD(t *testing.T) {
-	var scsiCount uint32 = 0
-	var vpmemCount uint32 = 1
-	var prfst uvm.PreferredRootFSType = uvm.PreferredRootFSTypeVHD
-	opts := &uvm.OptionsLCOW{
-		Options: &uvm.Options{
-			ID: "uvm",
-		},
-		VPMemDeviceCount:    &vpmemCount,
-		SCSIControllerCount: &scsiCount,
-		PreferredRootFSType: &prfst,
-		//ConsolePipe:         `\\.\pipe\vmpipe`,
-	}
+	opts := uvm.NewDefaultOptionsLCOW(t.Name(), "")
+	opts.SCSIControllerCount = 0
+	opts.VPMemDeviceCount = 1
+	opts.PreferredRootFSType = uvm.PreferredRootFSTypeVHD
```

Comment:
I think you need to pass rootfs.vhd for the root FS name, since you are changing the root FS type, and it doesn't automatically change anymore.